homeassistant/components/tuya/select.py

```diff
@@ -218,6 +218,26 @@
             device_class=DEVICE_CLASS_TUYA_FINGERBOT_MODE,
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
+    ),    # Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
+    "sd": (
+        SelectEntityDescription(
+            key=DPCode.SUCTION,
+            name="Suction",
+            entity_category=ENTITY_CATEGORY_CONFIG,
+            entity_registry_enabled_default=False,
```

Comment:
Why is this disabled by default?

Reply:
As there is an option in the vacuum UI I preferred to leave it disabled and the user to enable it if you want to use it in automations. Do you think it should be enabled?

Reply:
So if this is already part of the vacuum entity, why add it here again?